coralproject/talk
# [CORL-2392]: Add Comments and Users sub menu items to Moderation configure left nav

## What does this PR do?

These changes add `Comments` and `Users` as sub menu items under `Moderation` in the admin `Configure` tab. These changes are for https://vmproduct.atlassian.net/browse/CORL-2392.

See the [found documentation on navigation](https://github.com/4Catalyzer/found#navigation) for more details on the `Link` component and using it to render a custom button element, as is done in these changes.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

Go to the `Configure` tab in the admin. Click around the left nav, selecting different options, including `Moderation` and its new sub menu items, `Comments` and `Users`. See that clicking `Moderation` will take you to the moderation settings. Clicking `Comments` takes you to the comments-related moderation settings via anchor link, and clicking `Users` takes you to the users-related moderation settings via anchor link.
 
## How do we deploy this PR?
